org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/javadoc/HtmlToPlainText.java

```diff
@@ -0,0 +1,114 @@
+/**
+ * Copyright © 2009-2017, Jonathan Hedley <jonathan@hedley.net>
```

Comment:
I wasn't expecting to get away with it. We'll need to CQ JSoup anyway (dependency of Remark)